sequencer/src/main/java/net/consensys/linea/continoustracing/ContinuousTracer.java

```diff
@@ -53,7 +53,7 @@ public CorsetValidator.Result verifyTraceOfBlock(
     } catch (final Exception e) {
       throw new TraceVerificationException(blockHash, e.getMessage());
     } finally {
-      zkTracer.traceEndConflation();
+      // zkTracer.traceEndConflation();
```

Comment:
is it commented because not required anymore, or should be replace by something else?